src/meinberlin/meinberlin/static/stylesheets/scss/components/_comment_theme.scss

```diff
@@ -1,13 +1,80 @@
-.comment-main {
-    background: $color-background-base;
-    padding: 1em;
-    margin-bottom: 1em;
+.comment-listing {
+    .listing-element {
+        border: 0;
+    }
+}
+
+.comment-create-form,
+.comment-children-edit-form {
+    background: $color-background-base-introvert;
+    margin: 0;
+    padding: 0;
 
-    .comment-children & {
-        border-left: 1px solid $color-text-highlight-normal;
+    .comment-create-form-text,
+    .comment-edit-form-text {
+        margin: 0;
+        min-height: 70px / $font-size-normal * 1em;
     }
+
+    .form-footer-button {
+        font-size: $font-size-small;
+        line-height: 24px / $font-size-small * 1em;
+        padding: 0;
+        text-transform: capitalize;
+    }
+}
+
+.comment-header .comment-header-links {
+    margin-right: 1px;
+    padding: 0;
+
+    .comment-header-link {
+        @include icon-button;
+        margin: 0 -1px 0 0;
+    }
+
+    .comment-header-link-text {
+        @include squish-text;
```

Comment:
I don't like this because it does leave most users with only an icon, i.e. without a tooltip. Can we maybe change it? Have a `title` attribute in the HTML and use `:after { content: attr(title) }` in the core CSS?